modules/weko-deposit/weko_deposit/tasks.py

```diff
@@ -447,15 +470,28 @@ def _process(data_size, data_from):
             RecordIndexer().process_bulk_queue(
                 es_bulk_kwargs={'raise_on_error': True})
         if update_es_authorinfo:
+            weko_logger(key='WEKO_COMMON_IF_ENTER',
+                        branch="update_es_authorinfo is not empty")
             sleep(20)
-            for d in update_es_authorinfo:
+            weko_logger(key='WEKO_COMMON_FOR_START')
+            for i, d in enumerate(update_es_authorinfo):
+                weko_logger(key='WEKO_COMMON_FOR_LOOP_ITERATION',
+                        count=i, element=d)
+
                 dep = WekoDeposit.get_record(d['id'])
                 dep.update_author_link(d['author_link'])
+            weko_logger(key='WEKO_COMMON_FOR_END')
 
         data_total = search['hits']['total']
         if data_total > data_size + data_from:
+            weko_logger(key='WEKO_COMMON_IF_ENTER',
+                        branch="data_total is not empty")
```

Comment:
branch="data_total is not empty"
　=>  f"data_total > {data_size + data_from}"
               or  
          f"data_total is greater than {data_size + data_from}"